modules/buildkite-agent.nix

```diff
@@ -14,8 +14,16 @@
     openssh.publicKeyPath  = "/run/keys/buildkite-ssh-public";
     tokenPath              = "/run/keys/buildkite-token";
     meta-data              = "system=x86_64-linux";
-    hooksPath              = "/var/lib/buildkite-agent/hooks";
     runtimePackages        = with pkgs; [ gnutar gzip bzip2 xz ];
+    hooks.pre-command = ''
+      NIX_PROFILE_BINPATH=/run/current-system/sw/bin
```

Comment:
What are NIX_PROFILE_BINPATH and NIX_BUILD for?
Could the PATH be set up with `/run/current-system/sw/bin`?